### PR TITLE
clean the disable group and resource(SOFTWARE-3705)

### PR DESCRIPTION
**Pull request by yzz971115**

### CODE REVIEW

**Comment by brianhlin** [comment]
@edquist can you review this? https://opensciencegrid.atlassian.net/browse/SOFTWARE-3705

**Comment by edquist** [comment]
Yes i'll review.

**Comment by edquist** [comment]
Hi @yzz971115,

Looks like you have to manually fix a merge conflict for
`topology/University of Chicago/UChicago/OSG_UChicago.yaml`

I can assist if you need help with this.

File: topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml [hunk 1]
```diff
@@ -1,42 +1 @@
...
-  Description: One of OSSes became unavailable.
-  EndTime: May 29, 2017 14:00 +0000
-  ID: 1005324
-  ResourceName: UFlorida-SE
-  Services:
-  - SRMv2
-  Severity: Outage
-  StartTime: May 29, 2017 13:00 +0000
-# ----------------------------------------------------------
+
```
Comment (edquist) [inline topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml]: Looks like you replaced `topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml` with a single blank line -- but you need to remove the file.

Eg,
`git rm -f "topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml"`

File: topology/Fermi National Accelerator Laboratory/FermiGrid/FNAL_FERMIGRID_downtime.yaml [hunk 2]
```diff
@@ -1,26 +1 @@
...
-  EndTime: Oct 26, 2017 18:00 +0000
-  ID: 1005429
-  ResourceName: FNAL_GPGRID_5
-  Services:
-  - CE
-  - GridFtp
-  Severity: Outage
-  StartTime: Sep 18, 2017 14:00 +0000
-# ----------------------------------------------------------
+
```
Comment (edquist) [inline topology/Fermi National Accelerator Laboratory/FermiGrid/FNAL_FERMIGRID_downtime.yaml]: Same with `topology/Fermi National Accelerator Laboratory/FermiGrid/FNAL_FERMIGRID_downtime.yaml` -- you left a single blank line, but we just need to remove the file.

File: topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml (hunk 1 above)
Comment (brianhlin) [reply topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml]: That's my bad, I thought empty downtime files were ok

Comment (yzz971115) [reply topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml]: so all the empty _downtime file should be removed?

Comment (edquist) [reply topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml]: Yep!

Comment (yzz971115) [reply topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml]: So for the merge conflict with UChicago file, how should I do? I find it is a "Disable: true" group and just deleted this file.

Comment (edquist) [reply topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml]: It looks like the conflict for the UChicago file is from your f1195ae0ddaf891b671509e037066f9e24acc06a, which just removes the OSG VOOwnership.

Anyway, to resolve the conflict you can just remove (git rm -f) the file.

Comment (yzz971115) [reply topology/University of Florida/UF CMS Tier2/UFlorida-PG_downtime.yaml]: OK, I have removed that file in my previous branch

**Comment by brianhlin** [comment]
@yzz971115 it looks like the merge conflict still exists (see the GitHub PR page). Also, please only rebase and force-push if we request it, thanks!

**Comment by edquist** [comment]
Thanks @yzz971115 for the updates - I'll check it over

**Pull request merged**